javalib/src/main/scala/java/net/Socket.scala

```diff
@@ -130,8 +130,7 @@ class Socket protected (
   def bind(bindpoint: SocketAddress): Unit = {
     if (bindpoint != null && !bindpoint.isInstanceOf[InetSocketAddress]) {
       throw new IllegalArgumentException(
-        "Endpoint is of unsupported " +
-          "SocketAddress subclass"
+        "Endpoint is of unsupported SocketAddress subclass"
```

Comment:
Usually I would leave something like this as-is, unless
I had to change the file for other reasons.

If it is going to change, I suggest something like the
following flows better:
"Endpoint SocketAddress subclass is not supported".

It would be nice to match the text from a JVM, preferably OpenJDK, but this is probably not cost effective here.
Bigger fish.

Reply:
The boy scout rule would probably apply here. The concatenation of 2 string serves no purpose and is probably why you changed it in the first place. It makes the reader read the code more than once trying to understand why it is like that. Maybe go with Lee's new wording?